RUFAS/simulation_engine.py

```diff
@@ -257,6 +256,10 @@ def _initialize_simulation(self) -> None:
         crop_config_to_rufas_ids_map = self.field_manager.get_crop_configs_to_rufas_ids()
 
         nutrient_standard = NutrientStandard(self.im.get_data("config.nutrient_standard"))
+        # Uncomment the below three lines for testing pup
+        # self.im.delete_data("config.nutrient_standard")
+        # print(self.im.get_data("config.nutrient_standard"))
+        # print(self.im.get_metadata("properties.config_properties.nutrient_standard"))
```

Comment:
Just in case it's helpful I added a few lines to see the full effect before/after:

```
print(self.im.get_data("config.nutrient_standard"))
        print(self.im.get_metadata("properties.config_properties.nutrient_standard"))
        self.im.delete_data("config.nutrient_standard")
        print("data and metadata deleted")
        print(self.im.get_data("config.nutrient_standard"))
        print(self.im.get_metadata("properties.config_properties.nutrient_standard"))
```